### DISCUSSION TITLE
Clear transaction

### DISCUSSION THREAD

**Discussion by exiteden** [firefly-iii/firefly-iii#6057] [General]
Hello,

I am coming to Firefly III from GnuCash, and I need some help conceptually. I have always entered transactions manually and then, when I see they have hit my bank account, I "clear" them. This way I can see a "cleared balance" that matches my bank account balance, and a "total balance" that includes the transactions my bank balance doesn't see yet. I can't tell how to do this in Firefly.

As an example, if I write a check (my dentist still requires paper checks 🙄) I want to enter that transaction and see it in my "total balance" so I know that money is already spent, but I don't want that reflected in my "cleared balance" so that this balance matches my bank.

Is there a different way to think about this in Firefly? Am I doing money wrong?

Thanks for any help.

**Comment by JC5** [comment]
You can reconcile transactions to validate in hindsight, but you can't create them and "clear" them later.
